issue_05/script.js

```diff
@@ -9,3 +9,22 @@ const contents = [{
     alt: "画像2",
     text: "メッセージ"
 }];
+
+const ul = document.getElementById('js-ul');
+const promise = new Promise((resolve) => {
+    resolve(contents);
+}).then((value) => {
```

Comment:
```suggestion
const promise = new Promise((resolve) => {
    resolve(contents);
});
promise.then((value) => {

```
Nits
Since we're defining variables, it might be good to separate functions.

Reply:
As you said, it's easier to read if you separate them.
Thank you very much!
https://github.com/yoshikawa-akr/js-lessons/pull/5/commits/70dd55c45bd5ab222a3aac1a26b5cedd5aecd0a5